synapse/push/httppusher.py

```diff
@@ -294,8 +294,13 @@ def dispatch_push(self, event, tweaks, badge):
             defer.returnValue([])
         try:
             resp = yield self.http_client.post_json_get_json(self.url, notification_dict)
+            logger.info(
+                "Sent to: %s push-notify: %r get response: %r" % (
+                    self.url, notification_dict, resp
+                )
+            )
         except:
-            logger.warn("Failed to push %s ", self.url)
+            logger.warn("Failed to push %s %r" % (self.url, notification_dict))
```

Comment:
I'm slightly concerned that this will leak private information into the logs. Generally we've tried to keep such things out for INFO logging and greater. What do you think of putting the `notification_dict` in a separate DEBUG log line?